mux.go

```diff
@@ -14,16 +14,17 @@ import (
 
 // Config struct holds program configuration.
 var Config = struct {
-	Path       string
-	Listen     string
-	Log        string
-	CPUProfile string
-	TLSKey     string
-	TLSCert    string
-	TLS        bool
-	AppendOnly bool
-	Prometheus bool
-	Debug      bool
+	Path         string
+	Listen       string
+	Log          string
+	CPUProfile   string
+	TLS          bool
+	TLSKey       string
+	TLSCert      string
+	AppendOnly   bool
+	PrivateRepos bool
+	Prometheus   bool
+	Debug        bool
 }{
```

Comment:
Please do not revert this recent change and move TLS bool back up. Even if it's a slight optimization, I intend to follow linters suggestion and practice good struct packing. Thanks.